### PR TITLE
Add IoT class to RFLink integration

### PR DESCRIPTION
**Pull request by javicalle**
Add IoT class to RFLink integration. I consider this to be "Assumed State" and "Local Push"

"Assumed State" because of RF commands (switch, cover, light)
"Local Push" because of received RF signals (sensor and binary_sensor)

Ref: #14661

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by javicalle** [comment]
If/When approved I will fix the IoT class for all the RFLink devices.

File: source/_integrations/rflink.markdown [hunk 1]
```diff
@@ -4,6 +4,9 @@ description: Instructions on how to integrate RFLink gateway into Home Assistant
 logo: rflink.png
 ha_category:
   - Hub
+ha_iot_class:
+  - Assumed State
+  - Local Push
```
Comment (frenck) [inline source/_integrations/rflink.markdown]: This cannot be multiple.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/rflink.markdown (hunk 1 above)
Comment (javicalle) [reply source/_integrations/rflink.markdown]: I see.
Ok so I guess it is more appropriate to just leave the 'Assumed State' class.

**Review by frenck — APPROVED** [review]
Thanks, @javicalle 👍 

Happy Hacktoberfest 🎉

**Pull request merged**